python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -878,7 +878,7 @@ def commentTable(self, schema, tablename, comment=None):
         if comment is None:
             self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS NULL;'.format(schema, tablename))
         else:
-            self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS E\'{2}\';'.format(schema, tablename, comment))
+            self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS $${2}$$;'.format(schema, tablename, comment))
```

Comment:
```suggestion
            self._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS $escape${2}$escape$;'.format(schema, tablename, comment))
```

Just to play it super-safe?

Reply:
Seems good to me. It's at least cleaner :)